lighthouse-core/audits/redirects-http.js

```diff
@@ -17,7 +17,7 @@ class RedirectsHTTP extends Audit {
       title: 'Redirects HTTP traffic to HTTPS',
       failureTitle: 'Does not redirect HTTP traffic to HTTPS',
       description: 'If you\'ve already set up HTTPS, make sure that you redirect all HTTP ' +
-         'traffic to HTTPS. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/http-redirects-to-https).',
+         'traffic to HTTPS. [Learn more](https://web.dev/redirects-http).',
```

Comment:
this doc seems to give the opposite advice, we *want* them to redirect their http traffic to https and the doc is mostly talking about avoiding redirects